app/server.go

```diff
@@ -58,9 +76,11 @@ func main() {
 	httpsMux := mux.NewRouter()
 	httpsMux.HandleFunc(
 		"/heimdall/v1/token",
-		e.PostTokenHandler,
+		logging(strictTransportSecurity(e.PostTokenHandler)),
```

Comment:
You could do `httpsMux.Use(logging)` and `httpsMux.Use(strictTransportSecurity)`, but you'd have to modify the middleware functions. Might be more readable that way.